components/ui/SpringAnimation.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { animated, useSpring } from "react-spring";
+
+export const SpringAnimation: React.FC<{
```

Comment:
lets create a folder called animations inside of the ui folder and put all animations in there.